service/installer/helm/helm.go

```diff
@@ -205,6 +205,14 @@ func (i *HelmInstaller) runHelmCommand(name string, args ...string) error {
 		"stdout", stdOutBuf.String(), "stderr", stdErrBuf.String(),
 	)
 
+	if exiterr, ok := err.(*exec.ExitError); ok {
+		// if exit code is 137, than means it had been killed by kernel OOM.
+		// recreating the pod to avoid from reaching the memory limitation.
```

Comment:
```suggestion
		// If exit code is 137, then it had been killed by a kernel OOM.
		// Recreating the pod to prevent it reaching the memory limit.
```